hubblestack/utils/signing.py

```diff
@@ -124,10 +167,14 @@ def split_certs(fh):
             if line.startswith('----'):
                 ret = ret.encode()
                 try:
+                    log_level = log.debug
                     yield ossl.load_certificate(ossl.FILETYPE_PEM, ret)
-                except Exception as exception_object:
-                    log.debug('decoding item as certificate failed: %s; trying as PEM encoded private key',
-                        exception_object)
+                except Exception as E:
+                    status = STATUS.UNKNOWN
+                    if check_verif_timestamp(fh) == True:
```

Comment:
Redundant comparison to True, check_verif_timestamp() returns bool value already.